### PR TITLE
fix(Popconfirm): the title element does not have padding when icon is…

### PR DESCRIPTION
**Pull request by edc-hui**
… not set

[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [X] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#37346 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix: the title element does not have padding when icon is not set      |
| 🇨🇳 Chinese |     修复Popconfirm组件在没有icon的时候（包括rtl模式），padding依然存在的问题。      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before review. ⚠️

- [X] Doc is updated/provided or not needed
- [X] Demo is updated/provided or not needed
- [X] TypeScript definition is updated/provided or not needed
- [X] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
可以试试改成 flex 布局

**Comment by edc-hui** [comment]
@afc163 好的  我这边修改下

**Comment by edc-hui** [comment]
@afc163 您好，看下现在的修改方式可以嘛

File: components/popover/style/index.less [hunk 1]
```diff
@@ -90,22 +90,16 @@
   }
 
   &-message {
-    position: relative;
+    display: flex;
+    gap: 8px;
```
Comment (afc163) [inline components/popover/style/index.less:94]: gap 兼容性不好，用 flex。

Comment (edc-hui) [reply components/popover/style/index.less:94]: 好的  我这边在继续修改下

**Comment by edc-hui** [comment]
@afc163 您好，我这边又修改了一下。

**Comment by afc163** [comment]
挺好的。你还需要跑一下 `npm test -- -u` 更新一下 jest snapshot。

**Comment by afc163** [comment]
另外 dom 结构变了，发 feature 分支吧。

**Comment by edc-hui** [comment]
@afc163 好的  明白

**Comment by edc-hui** [comment]
由于rebase feature分支，导致分支代码错误。会重新提PR

**Pull request closed without merging**